### DISCUSSION TITLE
Author image size and position (just an idea)

### DISCUSSION THREAD

**Discussion by khannzxc** [razonyang/hugo-theme-bootstrap#145] [Ideas]
In mobile the author's image is very large, would it be good to use a compact height and width.

Now it looks like this:
![imagedell](https://user-images.githubusercontent.com/40522970/127135634-9e0309c3-bafd-415a-958d-c03dbfe0a495.JPG)

Use this instead:
![imagedell2](https://user-images.githubusercontent.com/40522970/127135680-be9cd19f-d0fb-4219-a791-8c433e34b4a2.JPG)

And for desktop also the same with the description below the image and name.

**Comment by razonyang** [comment]
LGTM, I may not have time to deal with it until this weekend.

**Comment by razonyang** [comment]
@khannzxc How about this style?
> Both desktop and mobile has the same layout.

![image](https://user-images.githubusercontent.com/17720932/127282372-c46cd151-0b13-4e88-bde7-ba2082e50dc0.png)

**Comment by razonyang** [comment]
@khannzxc You can now preview this in #147.

**Reply by khannzxc** [reply]
Yes Looking nice
